src/vdaf/prio3.rs

```diff
@@ -75,41 +75,48 @@ impl Prio3Aes128Count {
 /// The count-vector type. Each measurement is a vector of integers in `[0,2)` and the aggregate is
 /// the element-wise sum.
 #[cfg(feature = "crypto-dependencies")]
-pub type Prio3Aes128CountVec =
-    Prio3<CountVec<Field128, ParallelSum<Field128, BlindPolyEval<Field128>>>, PrgAes128, 16>;
+pub type Prio3Aes128SumVec =
+    Prio3<SumVec<Field128, ParallelSum<Field128, BlindPolyEval<Field128>>>, PrgAes128, 16>;
 
 #[cfg(feature = "crypto-dependencies")]
-impl Prio3Aes128CountVec {
-    /// Construct an instance of Prio3Aes1238CountVec with the given number of aggregators. `len`
-    /// defines the length of each measurement.
-    pub fn new_aes128_count_vec(num_aggregators: u8, len: usize) -> Result<Self, VdafError> {
```

Comment:
Reviewer note: It might be worth keeping this constructor around for convenience. (Similarly below.)